documentation/technical/metadata.md

```diff
@@ -67,6 +67,10 @@ This is for the version of the 360Giving Data Standard being used, so not the ve
 
 Find the details of the current version in the <a href="https://github.com/ThreeSixtyGiving/standard/releases" target="_blank" >Releases changelog</a>. 
 
+**Extensions**
+
+This is a text field that can be used to declare a list of extensions used in the file. Each extension is represented by a code, and multiple extensions can be declared by separating them with a semicolon `;` character.
```

Comment:
There was a conscious decision to not use a codelist for the extensions, so is using the term 'code' here potentially confusing?

As this is technical documentation and guidance, could we say "represented by a short string, which is a piece of text like a code"

Reply:
Good catch. Definitely agree on this.

Reply:
Changed via [0032e82](https://github.com/ThreeSixtyGiving/standard/pull/361/commits/0032e825a274cbea6733e99249611b261388edfc)